demo/admin/src/common/brevoModuleConfig/brevoContactsPageAttributesConfig.tsx

```diff
@@ -112,6 +113,7 @@ export const getBrevoContactConfig = (
         input2State: (values?: AdditionalFormConfigInputProps) => {
             return {
                 email: values?.email ?? "",
+                redirectionUrl: values?.redirectionUrl ?? "",
```

Comment:
shouldn't that be required?

or in this case throw an error if it is not set?

Reply:
It is required on create. But not on update. Both in api and admin.
Please check `packages/api/src/brevo-contact/dto/brevo-contact-input.factory.ts` and `packages/admin/src/brevoContacts/form/BrevoContactForm.tsx`